### PR TITLE
Updated dependencies and made global exception handling true

### PR DESCRIPTION
**Pull request by MahdiyarGHD**
### Checklist:

- [ ] It refers to an [Issue](https://github.com/EasyMicroservices/FileManager/issues).
- [ ] It fixes a bug.
- [ ] It's a new feature (non-breaking change which adds functionality)
- [ ] It's a breaking (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] CHANGELOG is updated.
- [ ] Related language label is selected.

### CODE REVIEW

File: src/CSharp/EasyMicroservices.ActionsMicroservice.Tests/CodeReviewerCheckTests.cs [hunk 1]
```diff
@@ -15,12 +15,12 @@ public class CodeReviewerCheckTests : CodeReviewerTests
         static CodeReviewerCheckTests()
         {
             //types to check (this will check all of types in assembly so no need to add all of types of assembly)
-            AssemblyManager.AddAssemblyToReview(
-                typeof(ApplicationManager),
-                typeof(StartUp),
-                typeof(LikeEntity),
-                typeof(LikeContract),
-                typeof(LikeController));
+            //AssemblyManager.AddAssemblyToReview(
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.ActionsMicroservice.Tests/CodeReviewerCheckTests.cs]: Do not comment these code please

File: src/CSharp/EasyMicroservices.ActionsMicroservice.WebApi/appsettings.Development.json [hunk 2]
```diff
@@ -7,10 +7,13 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "local": "Server=.;Database=Actions;Integrated Security=True;Trusted_Connection=True;TrustServerCertificate=True"
+    //"local": "Server=.;Database=Actions;Integrated Security=True;Trusted_Connection=True;TrustServerCertificate=True",
+    //"local": "Server=185.191.79.5,6752;Database=Actions;User ID=sa;Password=ijm4g83iq@;TrustServerCertificate=True;"
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.ActionsMicroservice.WebApi/appsettings.Development.json:11]: Please remove connection string

File: src/CSharp/EasyMicroservices.ActionsMicroservice.WebApi/appsettings.json [hunk 3]
```diff
@@ -7,7 +7,9 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "local": "Server=.;Database=Actions;Integrated Security=True;Trusted_Connection=True;TrustServerCertificate=True"
+    //"local": "Server=.;Database=Actions;Integrated Security=True;Trusted_Connection=True;TrustServerCertificate=True",
+    //"local": "Server=185.191.79.5,6752;Database=Actions;User ID=sa;Password=ijm4g83iq@;TrustServerCertificate=True;"
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.ActionsMicroservice.WebApi/appsettings.json:11]: Please remove connection string

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by Mohsen-Yaghoubi — APPROVED** [review]

**Review by Albert-the-jit — APPROVED** [review]

**Review by AmirmohammadNemati — APPROVED** [review]

**Review by mahdidelzendeh72 — APPROVED** [review]
Thanks

**Review by Ali-YousefiTelori — APPROVED** [review]

**Pull request merged**